### ISSUE TITLE
Is there any reason to use this repo over `bosh create-env` ?

### ISSUE THREAD

**Issue by evanfarrar** [cloudfoundry/bosh-init#131]
There are some prominent links[1] to this repo and I'm concerned that it should be mentioned that the bosh-init tool has basically been incorporated into bosh-cli v2, so we should fairly prominently in the README link to the official docs for that workflow[2].

I'm happy to PR the README with this notice, but was hoping for discussion to understand *if* folks still find this tool crucial and _why_. We wouldn't want to dissuade people from using this tool if there are use-cases that are still exclusively served by bosh-init

[1] https://en.wikipedia.org/wiki/BOSH_(software)#Deploying_BOSH
[2] https://bosh.io/docs/init.html

**Comment by dpb587-pivotal** [comment]
No, this repository should not be used, and it is not being maintained. Consumers should switch to the functionally-equivalent `bosh create-env` like you mentioned, if they haven't already.

Made [#153372223](https://www.pivotaltracker.com/story/show/153372223) to update this, but you might beat us to it with a pull request.

**Comment by evanfarrar** [comment]
I'll reserve the right to PR if I see folks getting confused in the short term, but otherwise it should just go into the queue.
